starport_template/lib/pages/assets_transfer_sheet.dart

```diff
@@ -41,24 +47,43 @@ class _AssetsTransferSheetState extends State<AssetsTransferSheet> {
                   SizedBox(height: theme.spacingXXXL),
                   Text('Transferring', style: CosmosTextTheme.title2Bold),
                   SizedBox(height: theme.spacingM),
-                  Text('This may take up to 1 minute.', style: CosmosTextTheme.copy0Normal),
+                  Text(
+                    'This may take up to 1 minute.',
+                    style: CosmosTextTheme.copy0Normal,
+                  ),
                   const Spacer(),
                   const ContentLoadingIndicator(),
                   const Spacer(),
-                  Text('1134.48 AKT', style: CosmosTextTheme.title0Medium),
+                  Text(
+                    '${formatAmount(widget.balance.amount.value.toDouble(), symbol: '')} ${widget.balance.denom}'
```

Comment:
we're loosing precision here, there is a reason why we went with `Decimal` objects instead of simply doubles. 

Here's short explanation: https://stackoverflow.com/a/3730040

you could try with:

```suggestion
                      '${widget.balance.amount.value.toString()}'
```
but if you need to control the precision, you can use `toStringAsPrecision()` then

cc: @wal33d006